src/core/migrations/markFactoriesAsDone.ts

```diff
@@ -0,0 +1,10 @@
+import type { RootState } from '@/core/zustand';
+import { produce } from 'immer';
+
+export function markFactoriesAsDone(state: RootState) {
+  return produce(state, draft => {
```

Comment:
I'll work on this, it's necessary to use the MigrationPlan as I wrote before since it's applied to saved game on load too